syncapi/storage/postgres/current_room_state_table.go

```diff
@@ -104,7 +104,7 @@ const selectEventsWithEventIDsSQL = "" +
 	// the rowsToStreamEvents expects there to be exactly six columns. We need to
 	// figure out if these really need to be in the DB, and if so, we need a
 	// better permanent fix for this. - neilalexander, 2 Jan 2020
-	"SELECT event_id, added_at, headered_event_json, 0 AS session_id, false AS exclude_from_sync, '' AS transaction_id" +
+	"SELECT event_id, added_at, headered_event_json, 0 AS session_id, false AS exclude_from_sync, '' AS transaction_id, 3 as history_visibility" +
```

Comment:
Please comment what's happening here. I'm guessing it's to ensure all current state events are visible?

Reply:
It's because `rowsToStreamEvents` is used in several places and, as the comment mentions, expects 7 columns and `current_room_state` doesn't have a `history_visibility` column.
We might need to query the actual `history_visibility` from the `output_room_events` table to get the correct value or would it be better to also add the column to the `current_room_state` table (or a join on `output_room_events`)?